libexec/bats-core/bats-preprocess

```diff
@@ -42,7 +42,7 @@ tests=()
       name="${name%[\'\"]}"
       body="${BASH_REMATCH[2]}"
       bats_encode_test_name "$name" 'encoded_name'
-      printf '%s() { bats_test_begin "%s"; %s\n' "$encoded_name" "$name" "$body" || :
+      printf '%s() { bats_test_begin "%s"; %s\n' "${encoded_name:?}" "$name" "$body" || :
```

Comment:
since `encoded_name` variable is not set explicitly and has a value returned by reference from `bats_encode_test_name()`, using `${x:?}` guarantees that the script will fail if the variable is not set.